docs/assistant/cloud/libraries/using-library-assets/add-single-items.md

```diff
@@ -0,0 +1,28 @@
+---
+id: add-single-items
+title: Adding single items to a site
+sidebar_label: Add single items
+---
+
+## Post types, blocks, Beaver Builder & 3rd party page builder templates
+
+* Navigate to the piece of content you’d like to import to your site from the Libraries App
+* Select Create and Edit or Import from the Create New drop-down menu
+	* Create and Edit will import the content and open the content for you to edit
```

Comment:
will import the content and open the content for you to edit-> imports the content and opens it for editing